_posts/2018-05-08-google-summer-of-code.markdown

```diff
@@ -13,8 +13,8 @@ categories:
 - ROS
 ---
 
-<img src="{{ site.url }}/assets/images/gsoc.png" width="750" style="margin-top:20px"/>
-
+>
```

Comment:
??

Reply:
now when u will create some post u dont have to put it inside content but you can use only media_link: http://wiki.ros.org/melodic?action=AttachFile&do=get&target=melodic.jpg